firefox-ios/nimbus-features/messaging/messaging-firefox-ios.fml.yaml

```diff
@@ -63,6 +63,9 @@ import:
               DEFAULT:
                 priority: 50
                 max-display-count: 5
+              MICRO_SURVEY:
+                priority: 50
+                max-display-count: 5
```

Comment:
IS there a reason for adding this, when `DEFAULT` is the same? I'm happy to add it, I guess I'm just questioning it's value.

Reply:
The value of having it separately is that we know that the style is specific to Micro Survey and can be adjusted without impacting messages that use the Default style. Currently, the values are arbitrary and can be changed, but most likely will be added on the experimenter side. However, I believe there's benefit in adding it here to also give more context on the micro survey feature for future developers.

Let me know if that justifies it, if not happy to discuss! :) thanks for questioning though, these conversations are definitely valuable!